faster_whisper/feature_extractor.py

```diff
@@ -1,4 +1,6 @@
 import numpy as np
+import torch
+import torchaudio.compliance.kaldi as ta_kaldi
```

Comment:
What kind of implications are there for using torchaudio as far as GPU RAM requirements?

Reply:
Torchaudio is used for feature extraction that happens on CPU ftb. Features are then passed to GPU for the model forward pass.
Note: Torchaudio can be made to work with GPU for faster feature extraction (I have a plan to do that). Since feature extraction is an initial step before model inference, it won't increase GPU memory usage.